metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_query.py

```diff
@@ -389,6 +391,9 @@ def usage_per_object_per_time_bucket_for_time_window(
         objects_column = (
             "BASE_OBJECTS_ACCESSED" if use_base_objects else "DIRECT_OBJECTS_ACCESSED"
         )
+
+        allowed_object_domains = f"('{SnowflakeObjectDomain.TABLE}', '{SnowflakeObjectDomain.EXTERNAL_TABLE}', '{SnowflakeObjectDomain.VIEW}', '{SnowflakeObjectDomain.MATERIALIZED_VIEW}')"
```

Comment:
It seems like we reuse the following sets of domains:
- Table, External table, View, Materialized view
- Table, External table

What do you think about putting them in variables, e.g.:
```
table_and_view_domains_filter = f"('{SnowflakeObjectDomain.TABLE}', '{SnowflakeObjectDomain.EXTERNAL_TABLE}', '{SnowflakeObjectDomain.VIEW}', '{SnowflakeObjectDomain.MATERIALIZED_VIEW}')"
table_domains_filter = f"('{SnowflakeObjectDomain.TABLE}', '{SnowflakeObjectDomain.EXTERNAL_TABLE}')"
```

Reply:
yeah, that would work, making this change now.

Reply:
done